kfdef/generic/istio/istio-noauth.yaml

```diff
@@ -16000,7 +15993,7 @@ spec:
 ---
 # Source: istio/charts/gateways/templates/deployment.yaml
 
-apiVersion: apps/v1
+apiVersion: extensions/v1beta1
```

Comment:
This can't be right.....This does not deploy at all on Kubernetes 1.16.
There may be a issue with selector....but not with apiVersion